bitnami/tomcat/templates/_pod.tpl

```diff
@@ -43,7 +43,7 @@ initContainers:
     mountPath: /bitnami/tomcat
 {{- end }}
 {{- if .Values.initContainers }}
-{{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) }}
+{{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 4 }}
```

Comment:
Hi @cache-sk,

Could you please apply the same fix used for the `.Values.sidecars` line?
```
 {{ include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) }}
```

The resulting indent using `nindent 4` is not technically bad:
```
      initContainers:
          - image: your-image
            imagePullPolicy: Always
            name: your-image-name
            ports:
            - containerPort: 1234
              name: portname
      containers:
      - name: tomcat
```

But for consistency reasons I would like the indent to be the same in the entire manifest.

Reply:
Done.